action.yml

```diff
@@ -0,0 +1,232 @@
+name: 'Setup wasixcc'
+description: 'Install wasixcc C/C++ compiler for the WASIX platform with optional toolchain components'
+author: 'wasix-org'
+
+branding:
+  icon: 'code'
+  color: 'blue'
+
+inputs:
+  version:
+    description: 'Version of wasixcc to install (e.g., v0.2.4, latest). Defaults to latest release.'
+    required: false
+    default: 'latest'
+  
+  sysroot-tag:
+    description: 'Tag of wasix-libc sysroot to install (e.g., v2025-01-01.1, latest). Leave empty to skip sysroot installation.'
+    required: false
+    default: 'latest'
```

Comment:
This should not be `leave empty to skip sysroot installation`. An empty string should not mean anything special. For not installing `install-sysroot` should be set to false.
Same goes for llvm and binaryen inputs